scripts/tasks/ts.ts

```diff
@@ -45,4 +47,7 @@ export const ts = {
     // Use default tsbuildinfo for this variant (since it's the only variant)
     return tscTask({ ...extraOptions, outDir: 'lib', module: 'commonjs' });
   },
+  build: () => {
+    return tscTask({ build: true });
+  },
```

Comment:
@ecraig12345 any thoughts on this ? I converted `react-codesandbox-northstar` to use tsc -b to be able to take a dep on the N* packages